azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java

```diff
@@ -287,11 +296,10 @@ protected FunctionCoreToolsHandler getFunctionCoreToolsHandler(final CommandHand
 
     protected void installExtension(final FunctionCoreToolsHandler handler,
                                     Set<BindingEnum> bindingEnums) throws Exception {
+        info(INSTALL_EXTENSIONS);
```

Comment:
why move this stat up?

Reply:
So that we will always show `Step 7 of 7: Installing function extensions if needed`
```
[INFO] Step 7 of 7: Installing function extensions if needed
[INFO] Extension bundle specified, skip install extension
```